src/20231203_AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum/AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203.t.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV2Ethereum} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IDefaultInterestRateStrategy} from 'aave-address-book/AaveV2.sol';
+
+import 'forge-std/Test.sol';
+import {ProtocolV2TestBase, ReserveConfig, InterestStrategyValues} from 'aave-helpers/ProtocolV2TestBase.sol';
+import {AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203} from './AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203.sol';
+
+/**
+ * @dev Test for AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203
+ * command: make test-contract filter=AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203
+ */
+contract AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203_Test is ProtocolV2TestBase {
+  AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18706339);
+    proposal = new AaveV2Ethereum_ChaosLabsRFAndIRUpdatesAaveV2Ethereum_20231203();
+  }
+
+  function _validateChanges(
+    ReserveConfig memory reserveConfigBefore,
+    ReserveConfig memory reserveConfigAfter,
+    ReserveConfig[] memory allConfigsAfter
+  ) internal view {
+    reserveConfigBefore.reserveFactor = 99_00;
+    reserveConfigBefore.interestRateStrategy = reserveConfigAfter.interestRateStrategy;
+    _validateReserveConfig(reserveConfigBefore, allConfigsAfter);
```

Comment:
This part should be after the "_validateInterestRateStrategy" as you override the "reserveConfigBefore.interestRateStrategy"